features/git-town/configuration/configuration_automatic.feature

```diff
@@ -9,10 +9,43 @@ Feature: Automatically running the configuration wizard if Git Town is unconfigu
     Given I haven't configured Git Town yet
 
 
-  Scenario Outline: Proceeding to configuration upon initial config prompt
+  Scenario Outline: Seeing a configuration prompt when running a Git Town command
```

Comment:
I'll call it prompt instead, since we haven't reached the wizard yet:

> Git Town hasn't been configured for this repository.
> Please run 'git town config --setup'.
> Would you like to do that now? [Y/n]